### PR TITLE
Ignore Report

### PR DESCRIPTION
**Pull request by joaopalopes24**

### CODE REVIEW

**Comment by joaopalopes24** [comment]
@stevebauman Check this PR please.

File: .scrutinizer.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ filter:
   excluded_paths:
     - tests/*
 build:
+  image: default-bionic
+  environment:
+      php: 8.1.2
```
Comment (stevebauman) [inline .scrutinizer.yml:7]: Not sure what this is -- we don't need it 👍 

Once removed I'll approve the PR.

```suggestion
```

**Review by stevebauman — CHANGES_REQUESTED** [review]

**Comment by joaopalopes24** [comment]
@stevebauman Check this docs: https://scrutinizer-ci.com/docs/guides/upgrading-trusty-image

File: .scrutinizer.yml (hunk 1 above)
Comment (stevebauman) [reply .scrutinizer.yml:7]: Oh I didn't see that Scrutinizer is now working with this -- thanks! 🙏

**Review by stevebauman — APPROVED** [review]

**Pull request merged**